includes/admin.php

```diff
@@ -54,13 +54,13 @@ function output_meta_box( WP_Post $post ) {
 	?>
 	<p>
 		<label for="hm_redirects_from_url"><?php esc_html_e( 'From URL', 'hm-redirects' ); ?></label><br>
-		<input type="text" name="hm_redirects_from_url" id="hm_redirects_from_url" value="<?php echo esc_attr( $post->post_title ); ?>" class="regular-text code"/>
+		<input type="text" class="widefat" name="hm_redirects_from_url" id="hm_redirects_from_url" value="<?php echo esc_attr( $post->post_title ); ?>" class="regular-text code"/>
```

Comment:
There is already a `class` attribute on the input tag, it's at the end of the line. So you need to adapt the existing class, instead of adding a new one.

Reply:
Ohh sorry. I completely missed the class attribute at the end.